app/mac/scripts/notarize.js

```diff
@@ -2,8 +2,8 @@ require('dotenv').config();
 const { notarize } = require('electron-notarize');
 
 exports.default = async function notarizing(context) {
-  const { electronPlatformName, appOutDir } = context;  
-  if (electronPlatformName !== 'darwin') {
+  const { electronPlatformName, appOutDir } = context;
+  if (electronPlatformName !== 'darwin' || process.env.CI === 'true') {
```

Comment:
Not related to this PR... but this reminds me: If you don't have the notarization files the app-mac build currently breaks. I'll make an issue/PR.

Reply:
You mean the notarization credentials, right? This line is runing them off for the CI. An alternative is that disable it if the APPLE* stuff used below is not set up.